### PR TITLE
newsletter 2019-06-14

### PR DESCRIPTION
**Pull request by ryan-shea**
📰 🗞 fresh and ready!! 🚂

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -949,22 +949,22 @@ def render_new_bounty_roundup(to_email):
...
-The Gitcoin blog is now live. <a href="https://gitcoin.co/blog/finding-best-freelance-jobs-for-coders-in-2019/">Take another look at our first post and send us any issues you may find!</a>
+As we prepare to enter the second half of 2019, we're asking our community for feedback. What are we doing well? What can we do better to help you reach your goals? <a href="mailto:team@gitcoin.co">Reach out to us if you'd like to share your thoughts.</a>
 </p>
 
 {kudos_friday}
 <h3>What else is new?</h3>
     <ul>
         <li>
-            Have an open source project that needs funding? <a href="mailto:founders@gitcoin.co">Send us an email and we'll see if we can help.</a>
+            Are you a project that might be interested in our hackathons? <a href="mailto:alex@gitcoin.co">Send us an email we'll be in touch.</a>
```
Comment (owocki) [inline app/retail/emails.py]: @alexvotofuture do you use this email address?

File: app/retail/emails.py [hunk 2]
```diff
@@ -933,7 +933,7 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "Hackathons on the Horizon!"
+    subject = "A Midsummer Bounty's Dream"
     new_kudos_pks = [3973, 2224, 1967]
```
Comment (owocki) [inline app/retail/emails.py]: would be great to rotate the kudos pks in the email here

File: app/retail/emails.py (hunk 1 above)
Comment (alexvotofuture) [reply app/retail/emails.py]: I have it forwarding to my other email, so should be good to use.

**Comment by ryan-shea** [comment]
email was sent

**Pull request closed without merging**